content/home/marketing/component-based.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: Component-Based
+title: కంపోనెంట్-బేస్డ్
 order: 1
 ---
 
-Build encapsulated components that manage their own state, then compose them to make complex UIs.
+వారి స్వంత స్థితిన్ని నిర్వహించే కప్పబడ్డ కంపోనెంట్లను రూపొందించండి, తరువాత సంక్లిష్ట UI లను రూపొందించడానికి వాటిని రూపొందించండి.
```

Comment:
I would suggest `ఇంకాప్సూల్టెడ్` or some other word instead of `కప్పబడ్డ`.

స్థితిన్ని -> స్టేట్ని
రూపొందించండి -> నిర్మించు
 
```
స్వంత స్టేట్ని  వాడుకునే  ఇంకాప్సూల్టెడ్ కంపోనెంట్లను నిర్మించండి, తరువాత కఠినమైన UI లను నిర్మించడానికి వాడండి
```